CHANGES.md

```diff
@@ -1,5 +1,55 @@
 # Changelog
 
+## 6.3 (2024-04-25)
+[View detailed changelog](https://github.com/spinalcordtoolbox/spinalcordtoolbox/compare/6.2...6.3)
+
+**FEATURE**
+ - **sct_deepseg**: Add CanProCo-based MS lesion segmentation model. [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4443)
+ - **sct_deepseg**: Add EPI-BOLD fMRI spinal cord segmentation model. [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4454)
+ - **sct_deepseg**: Update `contrast-agnostic` SC segmentation model to the latest v2.3 version. [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4451)
```

Comment:
I've preemptively added #4454 and #4460 to the changelog, so this PR should be up to date, barring any new PRs.